docs/UserGuide.md

```diff
@@ -136,6 +136,18 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd person in the address book.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
+### Deleting a module: `delmod`
+
+Deletes a moudule from FaculType.
```

Comment:
Typo on module

Reply:
Alright, I've changed it in the next commit.